src/query_parser/v2/parser.y

```diff
@@ -1145,6 +1115,16 @@ param_term(A) ::= ATTRIBUTE(B). {
   A.type = QT_PARAM_TERM;
 }
 
+param_term(A) ::= EMPTY_STRING(B) . [EMPTY_STRING] {
+  A = B;
+  A.type = QT_TERM_CASE;
+}
+
+param_term_case(A) ::= EMPTY_STRING(B) . [EMPTY_STRING] {
+  A = B;
+  A.type = QT_TERM_CASE;
+}
```

Comment:
If `EMPTY_STRING` ends up as a term, is there a point for the lexer to differentiate between `EMPTY_STRING` and `TERM`? can we pass from the lexer `TERM` in the case of `empty_string`?